docs/install/eks.md

```diff
@@ -0,0 +1,158 @@
+
+## Prerequisites
+
+Before creating an EKS cluster, make sure to:
+
+1. [Install the AWS Command Line Interface (AWS CLI)](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) for interacting with the different parts of AWS.
+
+2. [Install eksctl](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) to simplify cluster creation on EKS.
+
+3. [Install kubectl](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) to manage and deploy applications on Kubernetes.
+
+4. [Configure AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) with your credentials.
+
+## Create the EKS cluster
+
+1. Settled the required cluster details:
+
+    * name of your EKS cluster
+    * AWS region in which you wish to deploy your cluster
+    * the number of nodes you would like to have
+    * the desired ratio between [on-demand](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-on-demand-instances.html)
+        and [spot](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-spot-instances.html) instances in the total number of nodes.
+
+    !!! note
+
+        [spot](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-spot-instances.html) instances are not recommended for production environment but may be useful for testing purposes.
+
+2. Create your EKS cluster [following the official cluster creation instructions](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html).
+3. [Install the Amazon EBS CSI driver](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) on your cluster. See the [official documentation](https://docs.aws.amazon.com/eks/latest/userguide/managing-ebs-csi.html) on adding it as an Amazon EKS add-on.
+
+## Install the Operator and deploy your MongoDB cluster
+
+1. Deploy the Operator. By default deployment will be done in the `default`
+    namespace. If that's not the desired one, you can create a new namespace
+    and/or set the context for the namespace as follows (replace the `<namespace name>` placeholder with some descriptive name):
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl create namespace <namespace name>
+    $ kubectl config set-context $(kubectl config current-context) --namespace=<namespace name>
+    ```
+
+    At success, you will see the message that `namespace/<namespace name>` was created, and the context was modified.
+
+    To deploy the Operator [use](https://kubernetes.io/docs/reference/using-api/server-side-apply/) the following command:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl apply --server-side -f https://raw.githubusercontent.com/percona/percona-server-mongodb-operator/v{{ release }}/deploy/bundle.yaml
+    ```
+
+    ??? example "Expected output"
+
+        ``` {.text .no-copy}
+        customresourcedefinition.apiextensions.k8s.io/perconaservermongodbs.psmdb.percona.com serverside-applied
```

Comment:
```
backupstorages.everest.percona.com                   2023-09-19T11:15:54Z
databaseclusterbackups.everest.percona.com           2023-09-19T11:15:54Z
databaseclusterrestores.everest.percona.com          2023-09-19T11:15:55Z
databaseclusters.everest.percona.com                 2023-09-19T11:15:55Z
databaseengines.everest.percona.com                  2023-09-19T11:15:55Z
monitoringconfigs.everest.percona.com                2023-09-19T11:15:55Z
perconapgbackups.pgv2.percona.com                    2023-09-19T11:35:42Z
perconapgclusters.pgv2.percona.com                   2023-09-19T11:35:42Z
perconapgrestores.pgv2.percona.com                   2023-09-19T11:35:42Z
perconaservermongodbbackups.psmdb.percona.com        2023-09-19T11:16:49Z
perconaservermongodbrestores.psmdb.percona.com       2023-09-19T11:16:49Z
perconaservermongodbs.psmdb.percona.com              2023-09-19T11:16:49Z
perconaxtradbclusterbackups.pxc.percona.com          2023-09-19T11:24:07Z
perconaxtradbclusterrestores.pxc.percona.com         2023-09-19T11:24:07Z
perconaxtradbclusters.pxc.percona.com                2023-09-19T11:24:07Z
```
An idication of successfully provisioned EKS cluster

Reply:
Oh. I missed the point that it's not crd. we can attach an output of our successful CLI provisioning

Reply:
Removed everything following the Create the EKS cluster section